src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,16 @@
 import React from 'react';
+import { Comment } from '../../types/Comment';
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  comments: Comment[]
+};
+
+export const CommentList: React.FC<Props> = ({ comments }) => (
+  <div className="CommentList ui comments">
```

Comment:
```suggestion
  <div className="CommentList">
```
What for  they are?)

Reply:
in this project we have semantic ui, so I want to try it.

Reply:
Sorry, I`m not sure I see where U`r going with this(